frontend/src/views/PageNotFound.vue

```diff
@@ -1,65 +1,37 @@
 <template>
-  <v-container fluid>
-    <content-card
-      class="contentCard"
-      :title="$tc('views.pageNotFound.title')"
-      max-width="1200"
-      toolbar
-    >
-      <template #title-actions>
-        <v-btn
-          class="d-md-none"
-          icon
-          :to="{ name: 'profile', query: { isDetail: true } }"
-        >
-          <user-avatar :user="user" :size="36" />
+  <div class="sky">
+    <div class="hill">
+      <div class="d-flex absolute w-100 top-0 localnav">
+        <ButtonBack v-if="!$vuetify.breakpoint.mdAndUp" visible-label text dark />
+        <UserMeta v-if="!$vuetify.breakpoint.mdAndUp" avatar-only />
+      </div>
+      <ShootingStar class="shootingstar" />
+      <TentNight class="tent" />
+      <p class="white--text text-center px-3 mb-n8">
+        <i18n path="views.pageNotFound.detail">
+          <template #br><br /></template>
+        </i18n>
+      </p>
+      <div class="absolute bottom-0 justify-center d-flex col gap-2">
+        <v-btn v-if="user" text dark :to="{ name: 'home' }">
```

Comment:
Why only v-if=user? If I'm not logged in, I'm now completely lost in the intergalactic space without any place to visit 🙄 

Also, the button renders for me weirdly in the bottom-left corner:
<img width="1267" alt="Screenshot 2023-06-25 at 06 54 27" src="https://github.com/ecamp/ecamp3/assets/449555/edd12ef1-94f1-49f3-8813-94005b4768e0">

Reply:
Yep, just commited a fix